### PR TITLE
Add BitcoinCash Naming System (LNS)

### PR DESCRIPTION
**Pull request by mainnet-pat**
##### Short Description (to be shown on DefiLlama): 
LNS is ENS on smartBCH. Learn more: [http://bch.domains](https://t.co/Ehq1lIgDLz) | Register: http://app.bch.domains | Community & Support: [https://discord.gg/2Q52jB2Rzu](https://t.co/m1e6RHNbWh)

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is equal to the liquidity on the trading dex. LNS tokens locked in staking contract are counted towards staking.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/lns exports TVL: 
        
 
 
    smartbch                  107.86 k
    smartbch-staking          892.90 k
    staking                   892.90 k
    
    total                    107.86 k

File: projects/lns/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const WBCH = "0x3743eC0673453E5009310C727Ba4eaF7b3a1cc04";
+const WBCH_LNS_POOL = "0x7f3F57C92681c9a132660c468f9cdff456fC3Fd7";
+const CHAIN = "smartbch";
+
+module.exports = {
+  misrepresentedTokens: true,
+  methodology:
+    "TVL is equal to the liquidity on the trading dex. LNS tokens locked in staking contract are counted towards staking.",
+  smartbch: {
+    tvl: calculateUsdUniTvlPairs([WBCH_LNS_POOL], CHAIN, WBCH, [LNS], "bitcoin-cash", 18),
```
Comment (g1nt0ki) [inline projects/lns/index.js:15]: sorry, but you cant count entire LP as LNS tvl, this tvl belongs mistswap.

Staking part is fine.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The adapter at projects/lns exports TVL: 
        
 
 
    smartbch                  0
    smartbch-staking          891.70 k
    staking                   891.70 k
    
    total                    0

File: projects/lns/index.js (hunk 1 above)
Comment (mainnet-pat) [reply projects/lns/index.js]: Removed as you suggested

**Pull request merged**